src/revolve/db/postgres_adapter.py

```
@@ -0,0 +1,677 @@
+    def restore_schema_with_psycopg2(
+        self,
+        dump_file,
+        dbname,
+        user,
+        password,
+        host="localhost",
+        port=5432,
+        recreate_db=False
+    ):
+        if recreate_db:
+            self.recreate_database_psycopg2(self, dbname, user, password, host, port)
```

Pylint (3.3.7)

[error] Too many positional arguments for method call

(E1121)